src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferResults.cshtml.cs

```diff
@@ -138,10 +142,9 @@ public LocalOfferResultsModel(
     }
 
     public async Task<IActionResult> OnGetAsync(
-        string postcode, string? searchText, string? searchAge,
-        string? selectedLanguage, string? subcategorySelection,
-        string? costSelection, string? serviceDeliverySelection,
-        int? pageNum, bool forChildrenAndYoungPeople, Guid? correlationId
+        string postcode, string? searchText, string? searchAge, string? selectedLanguage,
```

Comment:
Things like this feel like an object should be passed in. I think I will suggest a code walkthrough with the team and review these things. No action.

Reply:
I do agree, some of these have a lot of arguments